.env.example

```diff
@@ -1,3 +1,3 @@
-VITE_RPC_URL= # Example: https://localhost:8545
-VITE_PROJECT_ID= # ProjectId from walletConnect
-VITE_ALCHEMY_KEY= # API key from Alchemy 
\ No newline at end of file
+NEXT_PUBLIC_RPC_URL= # Example: https://localhost:8545
+NEXT_PUBLIC_PROJECT_ID= # API from WalletConnect
```

Comment:
is this an api endpoint or a project id then?

Reply:
projectID, my bad...https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/a4a80bcb3d05cb213b045a2f40932f8ae160615c